components/admin/requests/Header.tsx

```diff
@@ -59,9 +63,64 @@ export default function RequestHeader({
         );
       case 'IN_PROGRESS':
         return (
-          <CompleteRequestModal applicationId={applicationId}>
-            <Button disabled={!allStepsCompleted}>Mark as complete</Button>
-          </CompleteRequestModal>
+          <Portal>
```

Comment:
Can we extract this to a component named `ProcessingFooter` and put it in `components/admin/requests`?